index.html

```diff
@@ -1031,4 +1060,5 @@ <h2>Sharing theme</h2>
       };
     }
   </script>
+  <script src="public/dictionary.js" async></script>
```

Comment:
👍 

I would like to investigate using [script type="module"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) (which implicitly defer) so we don't have globals floating around but I believe it's not trivial with Alpine